android/app/src/main/java/safetyfirst/androidapp/safetyfirstcontroller/MailBot/MailSender.java

```diff
@@ -1,5 +1,6 @@
 package safetyfirst.androidapp.safetyfirstcontroller.MailBot;
 
+//import java.net.PasswordAuthentication;
```

Comment:
Why is this commented out?

Reply:
Removed now, I used a different import with the same functionality in it's place as there were some issues relating to the commented out import.